### PR TITLE
Enable GitHub actions

### PR DESCRIPTION
**Pull request by nshalman**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -16,14 +16,13 @@ protos/cacher/cacher.pb.go: protos/cacher/cacher.proto
...
 run: ${server}
 	${server}
-test:
-	go test -race -coverprofile=coverage.txt -covermode=atomic ${TEST_ARGS} ./...
 else
 run: ${binaries}
 	docker-compose up --build server
-test:
-	docker-compose up test
 endif
```
Comment (mmlb) [inline Makefile:28]: lets get rid of this whole section too

**Review by mmlb — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (nshalman) [reply Makefile]: cleaned up.

**Review by mmlb — APPROVED** [review]

**Comment by mmlb** [comment]
you should delete the `.drone.yaml` file too

**Comment by nshalman** [comment]
> you should delete the `.drone.yaml` file too

Presumably that broke the drone runs. 🙃

**Comment by mmlb** [comment]
> > you should delete the `.drone.yaml` file too
> 
> Presumably that broke the drone runs. upside_down_face

Yep, since this PR is titled `Switch to GitHub actions` that implies no more drone right? :D. Do we want to do it in 2 phases? Add GHA, ensure both work, remove drone?

**Comment by nshalman** [comment]
> Yep, since this PR is titled `Switch to GitHub actions` that implies no more drone right? :D. Do we want to do it in 2 phases? Add GHA, ensure both work, remove drone?

Renamed to "Enable GitHub actions"

**Review by matoszz — APPROVED** [review]
I think we should work towards standardizing around some of the actions we use given nearly all our pipelines are identical, otherwise LGTM

**Pull request merged**